builder/chroot-script.sh

```diff
@@ -220,3 +225,13 @@ rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 echo "HYPRIOT_DEVICE=\"$HYPRIOT_DEVICE\"" >> /etc/os-release
 echo "HYPRIOT_IMAGE_VERSION=\"$HYPRIOT_IMAGE_VERSION\"" >> /etc/os-release
 cp /etc/os-release /boot/os-release
+
+# install Witty pi energy manager software
+echo "Installing i2c-tools"
+apt-get install i2c-tools
```

Comment:
See apt has no package lists at line 231 because they were cleaned up on line 222.
Instead you should add i2c-tools to line 150 (with escaping the prior newline).
I'll try to debug why our PR building didn't work again now.